src/tests/EdFi.DataManagementService.Tests.E2E/Features/Descriptors/DeleteDescriptorsValidation.feature

```diff
@@ -40,40 +40,40 @@ Feature: Resources "Delete" Operation validations
                   }
                   """
 
-        @ignore
         Scenario: 03 Verify error handling when trying to delete an item that has already been deleted
-            # The id value should be replaced with the resource created in the Background section
-             When a DELETE request is made to "/ed-fi/absenceEventCategoryDescriptors/{id}"
-             Then it should respond with 204
+            Given a DELETE request is made to "/ed-fi/absenceEventCategoryDescriptors/{id}"
              When a DELETE request is made to "/ed-fi/absenceEventCategoryDescriptors/{id}"
              Then it should respond with 404
 
-        @ignore
-        Scenario: 04 Verify response code when GET a deleted resource
-            # The id value should be replaced with the resource created in the Background section
-             When a DELETE request is made to "/ed-fi/absenceEventCategoryDescriptors/{id}"
-             Then it should respond with 204
+        Scenario: 04 Verify response code when trying to read a deleted resource
+            Given a DELETE request is made to "/ed-fi/absenceEventCategoryDescriptors/{id}"
              When a GET request is made to "/ed-fi/absenceEventCategoryDescriptors/{id}"
              Then it should respond with 404
 
-        @ignore
         Scenario: 05 Ensure clients cannot delete a descriptor that is being used by other Resources
-            Given the system has this "educationOrganizationCategoryDescriptors" descriptor
-                  | codeValue | namespace                                               | shortDescription |
-                  | School    | uri://ed-fi.org/EducationOrganizationCategoryDescriptor | School           |
```

Comment:
The team got rid of this way of creating descriptors. But we do still need to create the descriptor again, so that it is the last `id` value that we have recorded. Therefore I added a POST request, on line 57 below.